### ISSUE TITLE
env vars in shellac environments?

### ISSUE THREAD

**Issue by wizzard0** [geelen/shellac#11]
Of course it's possible to pass env vars via `export` etc, but this is prone to shell escaping/whitespace etc etc. Would be awesome to pass a dict of env vars to be mixed into the shellac environment, like the `.in()` currently does.

**Comment by geelen** [comment]
Definitely. Pretty easy for me to add stuff like this, but do you wanna give me a fuller example API? `.env()` maybe? Or something that would live inside the actual shellac block like `env ${ ... }`?

**Comment by wizzard0** [comment]
.env() sounds great
